submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/handlers/MetadataUpdateWebHandler.kt

```diff
@@ -0,0 +1,62 @@
+package ac.uk.ebi.biostd.submission.web.handlers
+
+import ac.uk.ebi.biostd.integration.SerializationService
+import ac.uk.ebi.biostd.submission.domain.helpers.SourceGenerator
+import ac.uk.ebi.biostd.submission.domain.service.ExtSubmissionService
+import ac.uk.ebi.biostd.submission.domain.service.SubmissionService
+import ac.uk.ebi.biostd.submission.model.SubmitRequest
+import ac.uk.ebi.biostd.submission.web.model.FileSubmitWebRequest
+import ebi.ac.uk.extended.mapping.to.toFile
+import ebi.ac.uk.extended.model.ExtFileList
+import ebi.ac.uk.extended.model.ExtSubmission
+import ebi.ac.uk.extended.model.FileMode.MOVE
+import ebi.ac.uk.extended.model.allFileList
+import ebi.ac.uk.extended.model.allFiles
+import ebi.ac.uk.model.FileList
+import ebi.ac.uk.model.Submission
+import ebi.ac.uk.model.SubmissionMethod
+import ebi.ac.uk.model.extensions.allSections
+import ebi.ac.uk.model.extensions.fileListName
+
+class MetadataUpdateWebHandler(
+    private val sourceGenerator: SourceGenerator,
+    private val webHandlerHelper: WebHandlerHelper,
+    private val submissionService: SubmissionService,
+    private val serializationService: SerializationService,
+    private val extSubmissionService: ExtSubmissionService
+) {
+    fun updateMetadata(request: FileSubmitWebRequest) {
```

Comment:
from what I see this operation will be as a slow as normal operation as all files will be re process (md5 calculated again)